src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -267,7 +236,7 @@ GetNativeCalendarName
 
 Gets the native calendar name.
 */
-CalendarDataResult
+ResultCode
```

Comment:
I re-ran the code formatter, and since this type name is now shorter, the rest of the method declaration got moved up to one line.